Yoast/developer
# Introduces the `lastmod` attribute instead of `priority` and `lastfreq` in the sitemap

Fixes #285 

## Summary
This PR introduces the `lastmod` attribute to the sitemap URLs, which replaces the `priority` and `lastfreq` attributes. With the update to Docusaurus 3 in #319, we can now make use of the implementation in https://github.com/facebook/docusaurus/pull/9954, as discussed in https://github.com/facebook/docusaurus/issues/2604.

## Relevant technical choices
Note that the configuration added to `docusaurus.config.js` will become a default in Docusaurus V4 and can then be removed.

## Test instructions
This PR can be acceptance tested by following these steps:
1. Open the `sitemap.xml`.
2. Confirm the URLs now don't have a `priority` and `lastfreq` attributes.
3. Instead, confirm most URLs have a `lastmod` attribute, which is the date they were last updated (in the GitHub repository).
4. Notably, this currently excludes the changelogs. We would need some custom code for that (TBD if that's of scope for this PR).

## Quality assurance
